docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -111,3 +111,4 @@ For an idea of how your source mappings may look like, refer to our [samples rep
 >
 > * There are no nuget.exe or dotnet.exe commands for managing the package source mapping configuration, see [NuGet/Home#10735](https://github.com/NuGet/Home/issues/10735).
 > * There are no means of mapping packages at package installation time, see [NuGet/Home#10730](https://github.com/NuGet/Home/issues/10730).
+> * There is a limitation when using the `DotNetCoreCLI@2` Azure Pipelines task which requires `feed-` prefixes in your source mapping configuration, see [microsoft/azure-pipelines-tasks#15542](https://github.com/microsoft/azure-pipelines-tasks/issues/15542).

```

Comment:
I think we should recommend using NuGetAuthenticate and not using that command instead. 

That's what the eng recommended in our issue: https://github.com/NuGet/Home/issues/11406#issuecomment-982096484

Reply:
Are we saying the same thing here but in opposite ways?

Reply:
updated just to clarify

Reply:
I think the wording of `requires you to do bla` implies that we want people to do it. 

It's not really a requirement imo, it's a bug. 

So I wouldn't call  out the whole `feed-` just say that PackageSourceMapping won't work with these tasks and tell them what to use.

Reply:
Are these comments to the outdated or most recent commit? I clarified majority of this already.

Reply:
The latest version. 

> * There is a limitation when using the `DotNetCoreCLI@2` Azure Pipelines task which requires `feed-` prefixes in your source mapping configuration, it is recommended to use `NuGetAuthenticate` for authentication needs instead, see [microsoft/azure-pipelines-tasks#15542](https://github.com/microsoft/azure-pipelines-tasks/issues/15542).


I don't think we should directly or indirectly suggest that people should add `feed-` to their nuget.config. 

My suggestion is to avoid that. 

Maybe: 

> * There is a limitation when using the `DotNetCoreCLI@2` Azure Pipelines task with Package Source Mapping. 
it is recommended to use `NuGetAuthenticate` for authentication instead, and use [Command Line Task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/utility/command-line) to invoke `dotnet`. See [microsoft/azure-pipelines-tasks#15542](https://github.com/microsoft/azure-pipelines-tasks/issues/15542).

You might need to replace the link if you go this direction.